resources/control_systems.md

```diff
@@ -5,66 +5,712 @@
+
+### 5.3 Obtaining Transfer Function from a Block Diagram
+
+**Recommended video:**
+> [Transfer Function from Block Diagram Reduction Example](https://youtu.be/7a2pyG1JkHQ?si=WuTTXtrO5CoSqGgn)
+
+### 5.4 MATLAB Implementation
+
+**Recommended video:**
 > [Transfer function model](https://www.mathworks.com/help/control/ref/tf.html)
 
-## 6 Control System Plots
+## 6 Time Response Analysis
+
+The output response of a system varying with time is the time response of the system.
+The time response of a system consists of a transient state and steady state. Below is
+a generic time response of a system with the 2 states.
+
+![](https://www.tutorialspoint.com/control_systems/images/time_response.jpg)
+
+Mathematically a systems response c(t) is written as
+
+$$c(t) = c_{tr}(t) + c_{ss}(t)$$
+
+### 6.1 Transient State Response
+
+This state encompasses a systems output response until it reaches a steady state. For
+large values of time _t_, the transient response will be zero.
+
+### 6.2 Steady State Response
+
+This state refers to the steady section of a systems response where the transient
+response reaches a zero value, as _t_ approaches infinity.
+
+### 6.3 Standard Test Signal
+
+#### 6.3.1 Unit Impulse Signal
+
+Mathematically a unit impulse, $\delta$ is defined as
+
+$$\delta = 0;~t\neq0$$
+
+![](https://www.tutorialspoint.com/control_systems/images/unit_impulse.jpg)
+
+#### 6.3.2 Unit Step Signal
+
+Mathematically a unit step signal, _u(t)_ is defined as
+
+$$u(t) = 1;~t\geq0$$
+
+$$~~~~~= 0;~t<0$$
+
+![](https://www.tutorialspoint.com/control_systems/images/unit_step.jpg)
+
+#### 6.3.3 Unit Ramp Signal
+
+Mathematically a unit step signal, _r(t)_ is defined as
+
+$$r(t) = t;~t\geq0$$
+
+$$~~~~~= 0;~t<0$$
+
+![](https://www.tutorialspoint.com/control_systems/images/unit_ramp.jpg)
+
+## 7 Response of A First Order System
+
+![](https://www.tutorialspoint.com/control_systems/images/unity_negative_feedback.jpg)
+
+From the first order system above, the transfer function is
+
+$$\frac{C(s)}{R(s)} = \frac{G(s)}{1 + G(s)}$$
+
+Substituting the value of _G(s)_
+
+$${C(s)} = \frac{1}{sT + 1}{R(s)}$$
+
+- ***_C(s)_*** is the laplace transform of output signal c(t)
+- ***_R(s)_*** is the laplace transform of input signal R(t)
+- ***_T_*** is the time constant
+
+For the time response of the output, the inverse laplace transform of the transfer function is performed.
+
+The table below shows the time domain responses to various signals. The constant values
+of the responses are the steady state components, while the transient components posses exponents.
+
+| Signal       | Response, _c(t)_                           |
+|--------------|--------------------------------------------|
+| Unit impulse | $$c(t) = \frac{1}{T}e^{\frac{-1}{T}}u(t)$$ |
+| Unit Step    | $$c(t) = (1-e^{\frac{-1}{T}})u(t)$$        |
+| Unit Ramp    | $$c(t) = (t-T-Te^{\frac{-1}{T}})u(t)$$     |
+
```

Comment:
Minor mistake of double line breaks. I believe an auto format will catch this however.